### PR TITLE
ADEN-3318 SourcePoint recovery

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AssetsManager/config.php [hunk 1]
```diff
@@ -829,6 +829,14 @@
 	)
 );
 
+$config['arecovery_lock_scss'] = array(
```
Comment (jacekjursza) [inline extensions/wikia/AssetsManager/config.php]: ARecoveryModule::ASSET_GROUP_ARECOVERY_LOCK ?

File: extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+class ARecoveryEngineHooks {
+	const ASSET_GROUP_ARECOVERY_LOCK = 'arecovery_lock_scss';
```
Comment (jacekjursza) [inline extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php]: What do you think about moving the constant to ARecoveryModule?

**Comment by jacekjursza** [comment]
One minor thing and LGTM. 
Also, the branch is currently conflicted.

**Comment by jacekjursza** [comment]
👍

**Comment by drets** [comment]
@fraszczakszymon can we remove `Don't merge` label?

**Pull request merged**